wrangler-service/pom.xml

```diff
@@ -87,6 +87,11 @@
       <artifactId>google-cloud-bigquery</artifactId>
       <version>${google.cloud.bigquery.version}</version>
     </dependency>
+    <dependency>
```

Comment:
shouldn't cloud-core be pulled by bigquery. I don't think we need it.

Reply:
Actually need it.. Compilation failed when it's not there:
```
cannot access com.google.cloud.RetryOption
[ERROR] class file for com.google.cloud.RetryOption not found
```